src/skvhttpclient/src/skvhttp/client/SKVClient.cpp

```diff
@@ -101,7 +101,7 @@ boost::future<Response<>> TxnHandle::write(dto::SKVRecord& record, bool erase, d
         .isDelete = erase,
         .precondition = precondition,
         .value = record.storage.share(),
-        .fieldsForPartialUpdate = std::vector<uint32_t>{}
+        .fieldsForPartialUpdate = std::vector<String>{},
```

Comment:
What is the reason  you want to use strings instead of indices? The k23si txn would translate these strings to indices anyway so might as well just pass the indices I think.

Reply:
I missed that there is a client Api that takes field indices. Switched to using indices.